### PR TITLE
Updates PGI C/C++ configurations

### PR DESCRIPTION
**Pull request by derobins**
Tweaks some settings and configures pgc++ to use C++11

### CODE REVIEW

File: config/pgi-cxxflags [hunk 1]
```diff
@@ -0,0 +1,97 @@
+#							-*- shell-script -*-
+#
+# Copyright by The HDF Group.
+# Copyright by the Board of Trustees of the University of Illinois.
```
Comment (lrknox) [inline config/pgi-cxxflags]:
```suggestion
```
line 4 should be removed since no part of this file existed before 2006.

**Review by lrknox — CHANGES_REQUESTED** [review]
Remove just the extra copyright line that doesn't apply

File: config/pgi-cxxflags (hunk 1 above)
Comment (derobins) [reply config/pgi-cxxflags]: Done

**Review by lrknox — APPROVED** [review]

**Review by byrnHDF — COMMENTED** [review]
What about CMake?

**Comment by derobins** [comment]
> What about CMake?

CMake sets the C++ standard the same way for all compilers, which is the fundamental problem being addressed here. I'll normalize the flags across platforms when the daily tests stabilize.

**Pull request merged**